packages/databases-collections/src/modules/data-service.js

```diff
@@ -28,7 +30,9 @@ export default function reducer(state = INITIAL_STATE, action) {
   if (action.type === DATA_SERVICE_CONNECTED) {
     return {
       error: action.error,
-      dataService: action.dataService
+      dataService: action.dataService,
+      configuredKMSProviders: action.dataService.configuredKMSProviders ? action.dataService.configuredKMSProviders() : [],
+      currentTopologyType: action.dataService.currentTopologyType ? action.dataService.currentTopologyType() : 'Unknown'
```

Comment:
Legit unsure how relevant it is here, but there is a global event in compass called `compass:deployment-awareness:topology-changed` that is used by I think at least one plugin to keep the topology type in sync with driver and I'm wondering if it's something that we need to keep in sync here too using this method

Reply:
@gribnoysup That’s a good question. Can you take a look at 3ecc6e1dd9124134236990e1ff4bb33c22808669 and see how okay that is?

I would rather use the `dataService` events we get from the source instead of global events, since that generally seems more reliable to me and more easily followable in terms of data flow, but I’m not sure if we do that elsewhere.